elements/pfe-tabs/src/pfe-tabs.scss

```diff
@@ -47,6 +47,7 @@ $pfe-tabs__panel--PaddingLeft-factor:  2;
   margin: 0 -1px 0 0;
   border: #{pfe-var(ui--border-width)} #{pfe-var(ui--border-style)} transparent;
```

Comment:
This would have to be a design-level discussion between CP & RHDC designers, but it seems to me like it might make sense to have the default tabs style include the border all the way around, like this https://xd.adobe.com/view/83f87db5-4e80-415e-4350-7b59ad213934-f5c9/screen/5e7c1ce3-7425-497a-b2ae-c0cf03ebd182/Web-1366-2?fullscreen

cc @markcaron

Reply:
@starryeyez024 Did you already add an issue for this in the PF Unified Design Kit repo? https://github.com/patternfly/patternfly-unified-design-kit

Reply:
@castastrophe added to this, thx! https://github.com/patternfly/patternfly-unified-design-kit/issues/59